docs/testing/playwright.mdx

```diff
@@ -76,4 +76,175 @@ Playwright is a well-established JavaScript testing framework. This guide aims t
   ```
 </Steps>
 
+## Test Helpers
+
+The `@clerk/testing` package also provides some helper functions to sign in/sign out with Clerk in your Playwright tests without having to interact with the UI.
+To use these commands, you just have to import the `clerk` object from the `@clerk/testing/playwright` package.
```

Comment:
Removing "just do X" to not make feel people who struggle with the instructions after it

```suggestion
To use these commands, import the `clerk` object from the `@clerk/testing/playwright` package.
```